src/protocol.h

```diff
@@ -355,8 +354,8 @@ enum GetDataMsg {
     MSG_FILTERED_BLOCK = 3,  //!< Defined in BIP37
     // Dash message types
     // NOTE: declare non-implmented here, we must keep this enum consistent and backwards compatible
-    MSG_TXLOCK_VOTE = 5,
     MSG_LEGACY_TXLOCK_REQUEST = 4,
+    /* MSG_TXLOCK_VOTE = 5, Legacy InstantSend and not used anymore  */
```

Comment:
Why not just remove instead of commenting it out to forget about later? :laughing:

Reply:
Hmm...I tend to prefer keeping such comments when there is a chance that a message with these types appears. Makes it easier to look up what's going on.

Reply:
:+1: